File: java assignment/NumberOfStudents.java
import java.util.Scanner;
public class NumberOfStudents{
	public static void main(String [] args){

		Scanner collect = new Scanner(System.in);

		int fails = 0;
		int passes = 0;
		
		for(int counter = 1; counter <=10; counter++){
		
			System.out.printf("Enter grade of student%d: ", counter);
			int scores = collect.nextInt();

			if(scores <= 45)
				fails += 1;

			else
				passes += 1;
		}

		System.out.printf("The number of student that passed is %d, The number of students that failed is %d", passes, fails); 

		
	}
}

File: pythonassignment/numberofstudents.py
passes = 0
fail = 0

for counter in range (1,11):
	scores = int(input(f'Enter student{counter} score: '))

	if scores <= 45:
		fail += 1

	else:
		passes += 1

print(f'The number of student that passed is {passes}, The number of students that failed is {fail}')
